docs/localization.md

```diff
@@ -1,5 +1,13 @@
 # Localization
 
+## UI Messages
+
+The master messages file is at `resources/master_messages.json`. It is encoded in the Chrome Apps JSON format. The translated files, consumed by the application's UI, live in `www/messages/<locale>.json`.
+
+## Transifex
```

Comment:
Just no need to mention Transifex, I guess?

Reply:
I thought about removing it, but I think it may be useful in the future?